product_seasonality/models/product_allowed_list_line.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2021 Camptocamp SA
+# @author: Julien Coux <julien.coux@camptocamp.com>
+# @author: Simone Orsi <simone.orsi@camptocamp.com>
+# @author: Damien Crier <damien.crier@camptocamp.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class ProductAllowedListLine(models.Model):
+    _inherit = "product.allowed.list.line"
+
+    date_start = fields.Datetime(required=True)
```

Comment:
not sure why we should keep this module as `product_seasonality`.
I'd rename this as `product_allowed_list_[date|weekly|calendar]`.

Reply:
hum I think the name `seasonality` reflects what it does but I can change if you want

Reply:
done.
renamed to `product_allowed_list_date`